swap/src/protocol/bob/message5.rs

```diff
@@ -66,7 +66,7 @@ impl Default for Behaviour {
         Self {
             rr: RequestResponse::new(
                 Codec::default(),
-                vec![(Message3Protocol, ProtocolSupport::Full)],
+                vec![(Message5Protocol, ProtocolSupport::Full)],
```

Comment:
Is `Full` actually correct here? Given that you are structuring it into Alice and Bob behaviours, this is likely either only inbound or outbound.